modules/tensor_mechanics/include/materials/SingleVariableReturnMappingSolution.h

```diff
@@ -64,15 +64,15 @@ class SingleVariableReturnMappingSolution
    * @param effective_trial_stress Effective trial stress
    * @param scalar                 Inelastic strain increment magnitude being solved for
    */
-  virtual Real computeResidual(const Real effective_trial_stress, const Real scalar) = 0;
+  virtual Real computeResidual(const Real & effective_trial_stress, const Real & scalar) = 0;
```

Comment:
Aie...this breaks a lot of my uncommitted branches, but it's necessary to template all creep models now, glad to see it in here, even if it is buried in a weird PR...